src/i18n.ts

```diff
@@ -0,0 +1,139 @@
+import { merge } from 'lodash';
+import i18n from 'i18next';
+import Moment from 'moment';
+import { extendMoment } from 'moment-range';
+import { initReactI18next } from 'react-i18next';
+import { registerLocale } from 'react-datepicker';
+import fr from 'date-fns/locale/fr';
+import km from 'date-fns/locale/km';
+import en from 'date-fns/locale/en-US';
+import { translation } from './config';
+import 'moment/locale/km';
+
+export const appResources = {
+  en: {
+    translation: {
+      // Date settings
+      date_locale: 'en',
+      Today: 'Today',
+      about: 'about',
+      'Tap the map to select': 'Tap the map to select',
+      'Click the map to select': 'Click the map to select',
+      Prism: 'Prism',
+      // Main buttons
+      Export: 'Export',
+      Legend: 'Legend',
+      // Menu
+      Hazards: 'Hazards',
+      Vulnerability: 'Vulnerability',
+      Exposure: 'Exposure',
+      Risk: 'Risk',
+      // Analysis
+      'Run Analysis': 'Run Analysis',
+      'Hazard Layer': 'Hazard Layer',
+      'Baseline Layer': 'Baseline Layer',
+      'Choose hazard layer': 'Choose hazard layer',
+      'Choose baseline layer': 'Choose baseline layer',
+      Statistic: 'Statistic',
+      Threshold: 'Threshold',
+      Above: 'Above',
+      Below: 'Below',
+      Mean: 'Mean',
+      Median: 'Median',
+      Date: 'Date',
+    },
+  },
+  // fr: {
```

Comment:
@ericboucher should i uncomment this or leave it there for now?

Reply:
@wadhwamatic can you think of a country that will use French? We can move that in a translation file for that country directly

Reply:
@ericboucher - we'll use French for the west / central africa deployment called 'rbd'

Reply:
@ericboucher regardless, if french is chosen, we will use location's English name instead of its local name, is that correct? if that's the case, in `isEnglishLanguageSelected` we will still need to check if the chosen language is English/French, therefore I'm still wondering if that's the best function name?

Reply:
No because the app's base language will still be English, so I think `isEnglishLanguageSelected` is a good name for now. We can adapt if needed

Reply:
done